docs/kbs.yaml

```diff
@@ -113,6 +114,16 @@ paths:
           schema:
             type: string
           required: true
+      requestBody:
+        required: false
+        description: >-
+          A KBS provisioned token that includes the Tee public key.
+          KBS will then use the Tee public key to perform JSON Web Encryption
+          for the resource in a response if the token is valid.
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/AttestationToken'
```

Comment:
I think we'd better put the token in the `bear_auth` header of HTTP Req. (Same as what we do in the `set-resource` API)

Reply:
Good point. Fixed.